### DISCUSSION TITLE
基于 XTuner 的大模型在线推理页面，支持多 Lora 切换

### DISCUSSION THREAD

**Discussion by limafang** [InternLM/Tutorial#370] [project]
基础功能
1. 基于 [PR 317](https://github.com/InternLM/xtuner/pull/317) 开发一个常见的 chat gradio
2. 实现多 Lora online 切换的功能，将 chat  gradio 扩展为，可以选择 lora，可以实现两个 lora 对比效果

**Comment by PrinceRay7** [comment]
您好，想加入，有位置吗，可以一起做吗

**Reply by vansin** [reply]
@PrinceRay7 可以的私聊 【浦语小助手微信号：InternLM】
